deploy-manage/deploy/elastic-cloud/upload-custom-plugins-bundles.md

```diff
@@ -248,4 +248,4 @@ https://api.elastic-cloud.com/api/v1/deployments/extensions \
 }'
 ```
 
-Please refer to the [Extensions API reference](https://www.elastic.co/docs/api/doc/cloud/group/endpoint-extensions) for the complete set of HTTP methods and payloads.
+See [Extensions API reference](https://www.elastic.co/docs/api/doc/cloud/group/endpoint-extensions) for the complete set of HTTP methods and payloads.
```

Comment:
Per [accessibility guidelines](https://docs.elastic.dev/tech-writing-guidelines/accessibility), `see` can be an ableist term. 
```suggestion
Refer to [Extensions API reference](https://www.elastic.co/docs/api/doc/cloud/group/endpoint-extensions) for the complete set of HTTP methods and payloads.
```